### ISSUE TITLE
Add icon for LFS files in file explorer

### ISSUE THREAD

**Issue by owochle** [go-gitea/gitea#30209]
### Feature Description

It would be nice if gitea had a way to quickly find out which files are stored on lfs. It's quite a small change but it could be very handy.

### Screenshots

![image](https://github.com/go-gitea/gitea/assets/62758885/40ad38de-b28d-4b86-8271-307a47e2dddb)
Hugging face example

**Comment by lunny** [comment]
To implement this, we need to store the LFS object's tree path in database.

**Comment by OwOchle** [comment]
Well, I don't know how it is implemented in the backend but lfs object are marked as so when viewing them :
![image](https://github.com/go-gitea/gitea/assets/62758885/f0ec8f5e-c715-4190-9259-b3fc4671c8ee)

**Comment by lunny** [comment]
For a special object, we can read the pointer. But for a file list(some directories have thousands files), reading the pointer is low performance.

**Comment by OwOchle** [comment]
Well, if possible it would be nice, but it's not that big of a deal, just handy. You may close the issue if you think it's too complex and/or useless to spend time on.

**Comment by lunny** [comment]
> Well, if possible it would be nice, but it's not that big of a deal, just handy. You may close the issue if you think it's too complex and/or useless to spend time on.

No, I think this is useful. Let's keep it open until someone would like to implement it.
